lib/Rex/Virtualization/Lxc.pm

```diff
@@ -0,0 +1,49 @@
+#
+# (c) Oleg Hardt <litwol@litwol.com>
```

Comment:
Please some one advise whether my name supposed to go here, or "(c) Jan Gehring <jan.gehring@gmail.com>"... i'm really not sure. 

thanks.